packages/esm-appointments-app/src/home-appointments/appointments-list.component.tsx

```diff
@@ -227,7 +241,7 @@ const AppointmentsBaseTable = () => {
           overflowMenuOnHover={isDesktop(layout)}
           rows={tableRows}
           size={isDesktop(layout) ? 'xs' : 'md'}
-          useZebraStyles={appointments?.length > 1}>
+          useZebraStyles={filteredAppointments?.length > 1}>
```

Comment:
Why isn't this just `{true}` ?

Reply:
I think it is because when it is one row, it resembles the headers but I think this is minor, I will just make it true.